src/nxt_cgroup.c

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (C) Andrew Clayton
+ * Copyright (C) F5, Inc.
+ */
+
+#include <nxt_main.h>
+#include <nxt_process.h>
+#include <nxt_fs.h>
+
+#include <stdio.h>
+#include <string.h>
+#include <limits.h>
+
+
+static nxt_int_t nxt_cgroup_create(const nxt_task_t *task,
+    nxt_process_t *process);
+
+
+static nxt_int_t
+nxt_cgroup_create(const nxt_task_t *task, nxt_process_t *process)
+{
+    char  cgpath[PATH_MAX];
+
+    if (task->thread->runtime->type != NXT_PROCESS_MAIN
+        || nxt_process_type(process) != NXT_PROCESS_PROTOTYPE
+        || process->isolation.cgroup.name == NULL)
+    {
+        return NXT_OK;
```

Comment:
This function only returns `NXT_OK`?

Reply:
Currently, but we may want to error check the nxt_fs_mkdir_all()

Reply:
Function no longer exists.